tests/scripts/check_names.py

```diff
@@ -832,12 +842,15 @@ def check_for_typos(self):
             for match
             in self.parse_result["public_macros"] +
             self.parse_result["internal_macros"] +
+            self.parse_result["private_macros"] +
             self.parse_result["enum_consts"]
             }
         typo_exclusion = re.compile(r"XXX|__|_$|^MBEDTLS_.*CONFIG_FILE$|"
-                                    r"MBEDTLS_TEST_LIBTESTDRIVER*")
+                                    r"MBEDTLS_TEST_LIBTESTDRIVER*|"
+                                    r"PSA_CRYPTO_DRIVER_TEST|"
+                                    r"PSA_CRYPTO_C")
```

Comment:
I know you are referring to https://github.com/Mbed-TLS/mbedtls/blob/531a871b888628be1dd30e84d137dae91aa51cfa/include/mbedtls/check_config.h#L330
I think it might not be a typo. The `MBEDTLS_` prefix is omitted from the macro names not only in `PSA_CRYPTO_C`, but also in `ECJPAKE_C` and `MD_C`. I would like to keep the exclusion here OR add the `MBEDTLS_` back for all these three macro names. What's you suggestion?

Reply:
Not really a typo, just a decision to shorten the names to keep it compact :) 
Compliance-(or typo-)checking scripts should have priority over such simplification in comments, so I would suggest either:
- Only changing `MBEDTLS_PSA_CRYPTO_C` to keep the script happy (it's fine, these are just comments);
- Changing all of these to full macro names to be consistent.